scrapy/utils/deprecate.py

```diff
@@ -134,7 +137,17 @@ def _clspath(cls, forced=None):
 DEPRECATION_RULES: List[Tuple[str, str]] = []
 
 
-def update_classpath(path):
+@overload
+def update_classpath(path: str) -> str:
+    ...
+
+
+@overload
+def update_classpath(path: Any) -> Any:
+    ...
+
+
+def update_classpath(path: Any) -> Any:
```

Comment:
Based on [the example on the docs](https://docs.python.org/3.7/library/typing.html#typing.overload), there should be no type hints here. Does mypy complain?

```suggestion
def update_classpath(path):
```

Reply:
If there is no type hints on the actual function, its body won't be checked (at least without `--check-untyped-defs`).